src/test/java/uk/gov/hmcts/dts/fact/model/admin/ContactTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.dts.fact.model.admin;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.dts.fact.entity.ContactType;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ContactTest {
+    private static final String TYPE = "type";
+    private static final String TYPE_CY = "type cy";
+    private static final String CONTACT_NUMBER = "123";
+    private static final String EXPLANATION = "explanation";
+    private static final String EXPLANATION_CY = "explanation cy";
+    private static final ContactType CONTACT_TYPE = new ContactType(10, TYPE, TYPE_CY);
+
+    @Test
+    void testCreationWhenEmailTypeIsSet() {
```

Comment:
Typo like mine 😉 with the email in the test name

Reply:
LOL. We like copying code around :)